db/schema.rb

```diff
@@ -213,6 +213,16 @@
     t.index ["lead_provider_id"], name: "index_provider_relationships_on_lead_provider_id"
   end
 
+  create_table "pupil_premium_eligibilities", id: :uuid, default: -> { "gen_random_uuid()" }, force: :cascade do |t|
```

Comment:
I think the table should just have been called PupilPremium. The eligibility part seems to be more about the operations we perform on the data rather than the data itself.

Reply:
The table contains %ages of pupils _eligible_ for pupil premium, hence the name. Happy to change it, I'm really not enjoying typing it out 😛